roles/openshift_aws/defaults/main.yml

```diff
@@ -61,6 +61,49 @@ openshift_aws_vpc:
 #    - cidr: 172.31.16.0/20
 #      az: "us-east-1a"
 
+openshift_aws_create_dns: False
+openshift_aws_dns_provider: "route53"
+# openshift_aws_dns_zone: ""
+# openshift_aws_dns_zone: "clusterid.example.com"
+# ie. openshift_aws_dns_zone: "{{ openshift_aws_clusterid }}.sysdeseng.com"
+
+openshift_aws_dns_records:
+  # Pertains to inventory file key: openshift_master_cluster_public_hostname
+  'api':
+    value: "{{ openshift_aws_elb_dict['master']['external']['name'] }}"
```

Comment:
rename this to 'elb_name' instead of value.

Add a new key 'value'
```yaml
value: "{{ elb_facts.elbs[0].dns_name }}"
```

Use this in route53 parameters directly.  This will allow us to specify other types and give real values (like a records, txt records, CNAME records that don't point to an elb, etc).